src/main/java/org/jenkinsci/plugins/proccleaner/PsCleaner.java

```diff
@@ -92,7 +91,7 @@ public PsCleanerDescriptor() {
         }
 
         public String getUsername() {
-            return username;
+            return System.getProperty("user.name", "");
```

Comment:
this is IMHO not correct: this sets the user name under which Jenkins master runs, not the slave (in case these two are different). If you want to get this information in run time (which is definitely better approach than the current one), you need to call it on the slave via channel, see e.g. [ProcCleaner#performCleanup](https://github.com/jenkinsci/proc-cleaner-plugin/blob/master/src/main/java/org/jenkinsci/plugins/proccleaner/ProcCleaner.java#L105)